.commitlintrc.json

```diff
@@ -1 +1,6 @@
-{ "extends": ["@commitlint/config-conventional"] }
+{
+  "extends": ["@commitlint/config-conventional"],
+  "rules": {
+    "body-max-line-length": [0]
```

Comment:
Why this change?

Reply:
Some old dependabot commits violate the `body-max-line-length` rule, which is set to 100 by `@commitlint/config-conventional`, but this is fixed when hardcoding the range of commits linted (using `--from` arg) (as seen in https://github.com/standard/ts-standard/pull/289#discussion_r1226158590).

This should probably be reverted if the `--from` argument of commitlint is hardcoded.